src/Common/src/System/Net/WebSockets/WebSocketValidate.cs

```diff
@@ -123,9 +124,19 @@ internal static void ThrowPlatformNotSupportedException()
 
         internal static void ValidateArraySegment(ArraySegment<byte> arraySegment, string parameterName)
```

Comment:
This change require additional tests for the WebSockets contract. Will this class behave similar to .Net Framework? What about WebSocket.Client?